CHANGELOG.md

```diff
@@ -8,6 +8,12 @@ Changelog](https://keepachangelog.com/en/1.1.0/), and this project adheres to
 
 ## [Unreleased]
 
+### Fixed
+
+- Fixes a bug in `migrate_with_timeouts` where passing a `stdout` parameter to
+  the command would raise a TypeError.
+  passed in.
```

Comment:
typo: "passed in." seems to have been left behind as part of editing that change message.